app/assets/javascripts/media/views/attachments/browser.js

```diff
@@ -0,0 +1,431 @@
+var View = techlogging.media.View,
+	mediaTrash = 'techlogging.media.view.settings.mediaTrash',
+	$ = jQuery,
```

Comment:
__[JSHint]__

Redefinition of '$'.


<!-- Comment by [SideCI](https://sideci.com) -->